tests/Model/Token/CustomerTest.php

```diff
@@ -13,7 +14,8 @@ public function validationProvider()
     {
         return [
             'Network address is empty' => [['networkAddress' => ''], 'Network address is empty'],
-            'Country code is empty' => [['networkAddress' => '93.174.192.154', 'countryCode' => ''], 'Country code is empty']
+            'Country code is empty' => [['networkAddress' => '93.174.192.154',
+                'countryCode' => ''], 'Country code is empty']
```

Comment:
Nitpick: consider reformatting array to the following for easier readability:
```php
    'Country code is empty' => [
        [
            'networkAddress' => '93.174.192.154',
            'countryCode' => '',
        ],
        'Country code is empty',
    ]
```

Reply:
True, auto formatter does sometimes weird things. Will fix